src/Raven.Server/Smuggler/Documents/Processors/IndexProcessor.cs

```diff
@@ -225,6 +231,16 @@ private static IndexDefinition ReadLegacyIndexDefinition(BlittableJsonReaderObje
             return indexDefinition;
         }
 
+        private static string ReplaceLegacyProperties(string str)
+        {
+            if (string.IsNullOrWhiteSpace(str))
+                return str;
+
+            str = str.Replace("new Raven.Abstractions.Linq.DynamicList", string.Empty);
```

Comment:
I think it should be changed to 'DynamicArray' ?
Check here:
https://github.com/ravendb/ravendb/blob/v4.1/src/Raven.Client/Documents/Indexes/ExpressionStringBuilder.cs#L1464

Reply:
in v3.x we had: https://github.com/ravendb/ravendb/blob/v3.5/Raven.Client.Lightweight/Indexes/ExpressionStringBuilder.cs#L1588

Reply:
done